docs/api/server/CLI-Commands.md

```diff
@@ -0,0 +1,204 @@
+
+| Flag       | Description                                                                                    |
+|---------------|------------------------------------------------------------------------------------------------|
+| `--root-module-name`  | Name of the [Holocron Module] that serves as the entry point to your application. |
+| `--module-map-url`  | Remote Module Map URL for [`one-app-dev-cdn`] to proxy. *For Development.* |
+| `--use-middleware` | Apply a custom middleware configuration for [`one-app-dev-proxy`]. *For Development.* |
+
+#### Environment Variables
+
+Please see the [Environment Variables](./Environment-Variables.md) API docs.
+
+## Test Commands
+
+| Command       | Description                                                                                    |
+|---------------|------------------------------------------------------------------------------------------------|
+| `test` | Runs all `test:*` scripts. |
+| `test:unit` | Runs all [Jest] Unit Tests. Accepts Jest flags. |
+| `test:lint` | Runs [ESLint] [rules](https://github.com/americanexpress/eslint-config-amex) against source. |
+| `pretest:integration` | Using [`concurrently`], runs `build:prod-sample` and pulls Docker containers to prepare the [Integration Test Suite]. |
+| `test:integration` | Executes the [Integration Test Suite] using [Jest]. |
+| `test:danger` | Runs the [DangerJS tool](https://danger.systems/js/) that returns bundle size stats and a preflight checklist for PRs. |
+| `test:git-history` | Validates the Git commit message format using the [`commitlint`](https://github.com/conventional-changelog/commitlint) tool. |
+| `test:lockfile` | Validates that the `package-lock.json` file contains valid NPM registry URLs using the [`lockfile-lint`](https://github.com/lirantal/lockfile-lint) tool. |
+| `prestart:prod-sample` | Runs [`build:sample-modules`](#build-commands) before `start:prod-sample`. |
+| `start:prod-sample` | Runs a [Docker Compose](https://docs.docker.com/compose/compose-file/) file to start up the [Integration Test Suite] containers. |
+
+## Development Commands
+
+> 💬 These commands only work in Development
+
+| Flag | Description |
+|------|-------------|
+| `serve-module` | Accepts a path to a [Holocron Module] and adds it to the local development CDN. *See [`serve-module` Arguments](#serve-module-arguments).*  |
+| `drop-module` | Accepts a [Holocron Module] name and removes the Module from the local development CDN. *See [`drop-module` Arguments](#drop-module-arguments).* |
+| `set-middleware` | Accepts a Node JS file that defines custom [Express Middleware](http://expressjs.com/en/guide/writing-middleware.html#writing-middleware-for-use-in-express-apps) for the development proxy. *See [`set-middleware` Arguments](#set-middleware-arguments).* |
+| `set-dev-endpoints`| Accepts a Node file containing development endpoints for the development proxy. *See [`set-dev-endpoints` Arguments](#set-dev-endpoints-arguments).* |
```

Comment:
> Accepts a Node file containing development endpoints for the development proxy

Can we elaborate on the development proxy a little bit more pls ... just a little detail on what it does. 

Somewhere in the md: maybe in a notes section or within some parenthesis when we first mention it

Reply:
> Can we elaborate on the development proxy a little bit more pls ... just a little detail on what it does.
This CLI Commands file is purely a reference file and I was cautioned from going into detail on those libraries and leave it to the "Development-Tools.md" effort.

Reply:
how about hyperlinking to the github repos (or the `Development-Tools.md`, whichever gives better context) when you first declare both `local development cdn` and `development proxy`?